Foundation/URLSession/http/HTTPURLProtocol.swift

```diff
@@ -557,7 +558,7 @@ extension _HTTPURLProtocol: _EasyHandleDelegate {
             completeTask()
         case .failWithError(let errorCode):
             internalState = .transferFailed
-            failWith(errorCode: errorCode, request: request)
+            failWith(errorCode: errorCode, errorDescription: "Completion failure", request: request)
```

Comment:
We're in a somewhat unfortunate place here, because localization doesn't quite work yet, but we still should find a way to make this use the usual localized string lookup API (on bundle) instead of hard coding English strings.

Reply:
I’ve created a placeholder for a future NSLocalizedString(_, _) implementation and used it.